CHANGELOG.md

```diff
@@ -14,4 +14,8 @@ Initial release.
 ### Added
 - OrchardBundle enum
 - first_issuance flag
-- action-group-based serialization
\ No newline at end of file
+- action-group-based serialization
+
+## [0.3.0] - 2025-05-05
```

Comment:
Please suggest if a new version is needed, and if it needs to be a major version bump

Reply:
no, you mark it as [Unreleased]

Reply:
Done